### ISSUE TITLE
glmer not found

### ISSUE THREAD

**Issue by brgew** [lme4/lme4#658]
Hi,

We are using the `lme4::glmer.nb` function in a package and noticed unexpected behavior.

We load our package and call `lme4::glmer.nb` as

`res <- lme4::glmer.nb(model_formula, nAGQ=0, control=lme4::glmerControl(optimizer = "nloptwrap"), offset = log(Size_Factor), ...)`

The function appears to return immediately (and silently) without running, as best I can tell. Adding `verbose=TRUE` gives the message

`<simpleError in glmer(model_formula, nAGQ = 0, control = lme4::glmerControl(optimizer = "nloptwrap"),     offset = log(Size_Factor), family = stats::poisson, verbose = FALSE): could not find function "glmer">`

The function appears to run correctly when we either load+attach lme4 using `library(lme4)` or by using '`@importFrom lme4 glmer`' in our function header. 

I wonder whether this is expected.

Thank you.

**Comment by bbolker** [comment]
No, it's not.

Simpler reproducible example:

```r
set.seed(101)
dd <- data.frame(x=runif(200), f= rep(1:20, each=10))
b <- rnorm(20)
dd <- transform(dd, y = rnbinom(200, mu  = exp(1 + 2*x + b[f]), size = 2))
lme4::glmer.nb(y~x + (1|f), data = dd)
```

**Comment by bbolker** [comment]
This is now fixed in the development version of `lme4`, although since we just sent in a new release to CRAN it may be a little while before this makes it to CRAN. How critical is it to your use case that the fix gets onto CRAN quickly?

The other issue stopping me from closing this right away is that it's a little bit tricky to test (we have to make sure to test in an environment that does *not* have `lme4` loaded.)

**Comment by bbolker** [comment]
fixed (tested) in 31d1d0a
